helm/dagster/values.yaml

```diff
@@ -1124,6 +1124,8 @@ dagsterDaemon:
   #   - name: secret
   envSecrets: []
 
+  envList: []
```

Comment:
could also be podEnv or containerEnv since it goes directly on the pod instead of configmap -> pod? But that seems more confusing